lib/cn/cn.c

```diff
@@ -459,54 +448,28 @@ cn_mblocks_commit(
  * abort the remaining N-@n_committed.
  */
 static void
-cn_delete_blks(struct mpool *mp, struct blk_list *blks, u32 *const n_committed)
+cn_delete_blks(struct mpool *mp, struct blk_list *blks)
 {
-    u32 bx;
-
-    for (bx = 0; bx < blks->n_blks; ++bx) {
-        if (*n_committed > 0) {
-            delete_mblock(mp, &blks->blks[bx]);
-            *n_committed -= 1;
-        } else {
-            abort_mblock(mp, &blks->blks[bx]);
-        }
-    }
+    for (uint32_t bx = 0; bx < blks->n_blks; ++bx)
```

Comment:
Can you call the delete_mblocks() variant here?

Reply:
I would probably just change bx to i, like you do elsewhere...

Reply:
Great catch!